packages/ui-editable/src/InPlaceEdit/styles.js

```diff
@@ -48,7 +48,6 @@ const generateStyle = (componentTheme, props, state) => {
       touchAction: 'manipulation',
       background: 'transparent',
       border: 'none',
-      fontFamily: componentTheme.fontFamily,
```

Comment:
Why was this deleted?

Reply:
Because there was no corresponding themeVariable set for it in the theme.js. I looked up the PR from before, remembered we had a discussion about it: https://github.com/instructure/instructure-ui/pull/328. The fontFamily was deleted from the theme.js then, so this can be deleted too.

Reply:
I see, thanks!